packages/wrangler/src/dev/miniflare.ts

```diff
@@ -61,6 +76,7 @@ function createClass({ className, proxyUrl }) {
 			proxyRequest.headers.set(HEADER_URL, request.url);
 			proxyRequest.headers.set(HEADER_NAME, className);
 			proxyRequest.headers.set(HEADER_ID, this.id);
+			proxyRequest.headers.set(HEADER_CF_BLOB, JSON.stringify(request.cf));
```

Comment:
Is everything in `request.cf` JSON-stringify-able? I'm guessing yes, but just double-checking.